test/test_gaussjacobi.jl

```diff
@@ -56,7 +56,7 @@ x, w = gaussjacobi(10013, .9, -.1)
 # test last alpha and beta parameters:
 x, w = gaussjacobi(100, 19., 21.)
 @test abs(x[87] - 0.832211446176040) < tol
-@test abs(w[50] - 0.064530500882703) < tol
+@test abs(w[50] - 0.026363584978877) < tol
```

Comment:
## On FastGaussQuadrature `v0.2.1`:

```julia
julia> versioninfo()
Julia Version 0.6.4
Commit 9d11f62bcb (2018-07-09 19:09 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: AMD Ryzen 7 2700X Eight-Core Processor
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Prescott MAX_THREADS=16)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, generic)

julia> using FastGaussQuadrature

julia> x,w = gaussjacobi(100,19.,21.)
([-0.975714, -0.966612, -0.95738, -0.947717, -0.937528, -0.926775, -0.91544, -0.903517, -0.891005, -0.877906  …  0.887979, 0.900454, 0.91233, 0.923604, 0.934276, 0.944349, 0.953838, 0.962773, 0.971233, 0.979473], [1.17239e-30, 8.37369e-28, 1.33881e-25, 9.36954e-24, 3.76416e-22, 1.00338e-20, 1.94014e-19, 2.88923e-18, 3.45836e-17, 3.43497e-16  …  1.68918e-14, 1.96211e-15, 1.9022e-16, 1.49211e-17, 9.08337e-19, 4.04761e-20, 1.21017e-21, 2.10752e-23, 1.64082e-25, 2.97148e-28])

julia> w[50]
0.06453050088270522
```

## On FastGaussQuadrature `v0.4.7`:

```julia
julia> versioninfo()
Julia Version 1.5.3
Commit 788b2c77c1 (2020-11-09 13:37 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: AMD Ryzen 7 2700X Eight-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, znver1)

julia> using FastGaussQuadrature
[ Info: Precompiling FastGaussQuadrature [442a2c76-b920-505d-bb47-c5924d526838]

julia> x, w = gaussjacobi(100,19.,21.)
([-0.9757141397469644, -0.9666121240571874, -0.9573798840365513, -0.9477167985263666, -0.9375280986658256, -0.926774886572364, -0.9154401126020022, -0.9035171506477786, -0.8910050589146083, -0.8779063259024655  …  0.8879790740356485, 0.9004541569915283, 0.912330333845481, 0.9236043594923872, 0.9342758802090618, 0.9443493100942537, 0.9538377996373524, 0.9627728626813408, 0.9712332364559879, 0.9794730128994685], [4.789727417487866e-31, 3.421026256989471e-28, 5.469648476341766e-26, 3.8278743775992884e-24, 1.5378280253776293e-22, 4.099274050284742e-21, 7.926345087673299e-20, 1.1803786331462114e-18, 1.412892875478951e-17, 1.403336363468946e-16  …  6.9010586047924e-15, 8.016080229236447e-16, 7.77135148594776e-17, 6.0959320745298726e-18, 3.7109616413800584e-19, 1.653628648369213e-20, 4.944076669666898e-22, 8.610147653206262e-24, 6.703478067293602e-26, 1.2139804389168331e-28])

julia> w[50]
0.026363584978877218
```

## Conclusion
The original issue seems already solved.